TidepoolOnboarding/Support/ValidDeviceDetector.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ValidDeviceDetector.swift
```

Comment:
Consider `JailbrokenDeviceDetector` since `Valid` could mean a lot of different things.